tasks/Task.cpp

```diff
@@ -234,7 +234,12 @@ void Task::errorHook()
 }
 void Task::stopHook()
 {
-    driver->resetDevice(SEND_BUFFER);
+    driver->resetDevice(SEND_BUFFER, true);
+    if(state() == EXCEPTION)
```

Comment:
I thought it would make sense, given the exception we're having. But now I'm not sure, other exceptions could happens.